static/js/formatters-internal.js

```diff
@@ -525,24 +525,24 @@ export function price(fieldValue = {}, locale) {
 
 /**
  * Returns a localized price range string for the given price range ($-$$$$) and country code (ISO format)
- * @param {Object} pricerange The price range from LiveAPI entity
+ * @param {string} defaultPriceRange The price range from LiveAPI entity
  * @param {string} countrycode The country code from LiveAPI entity (e.g. profile.address.countryCode)
  * @return {string} The price range with correct currency symbol formatting according to country code
  */
-export function priceRange(priceRange, countryCode) {
-  if (!priceRange || !countryCode) {
+export function priceRange(defaultPriceRange, countryCode) {
+  if (!defaultPriceRange || !countryCode) {
     console.warn(`No price range or country code given.`);
     return '';
   }
   const currencyCode = LocaleCurrency.getCurrency(countryCode);
   if (currencyCode) {
     const currencySymbol = getSymbolFromCurrency(currencyCode);
     if (currencySymbol) {
-      return Array(priceRange.length+1).join(currencySymbol); 
+      return Array(defaultPriceRange.length+1).join(currencySymbol); 
```

Comment:
Could we just do defaultPriceRange.replaceAll('$", currencySymbol) instead? Seems a bit simpler.

Reply:
hm you are right. I guess the backend will only have $ for now. I will update this!